CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.4.2 (2024-03-11)
+
+
```

Comment:
If the only change here is internal/CI, then we can skip this release.

Reply:
Sounds good to me. I will close the PR and delete the branch.